src/utils/isIgnored.spec.ts

```diff
@@ -0,0 +1,83 @@
+import * as fileModule from '@sasjs/utils/file'
+import * as getProjectRootModule from './getProjectRoot'
+import * as getLintConfigModule from './getLintConfig'
+import { DefaultLintConfiguration } from './getLintConfig'
+import { LintConfig } from '../types'
+import { isIgnored } from './isIgnored'
+
+describe('isIgnored', () => {
```

Comment:
we should also test if filepath e.g. `sasjs/common/init/init.sas` is ignored if there is an entry to ignore `sasjs/common`.

Reply:
done